index.html

```diff
@@ -212,17 +212,17 @@ <h5 class="font-weight-bold text-white mb-lg-4 pb-2 margin-top-mobile">Features<
                 </div>
                 <div class="col-lg-3 col-md-3 col-sm-6 col-12">
                     <h5 class="font-weight-bold text-white mb-lg-4 pb-2 margin-top-mobile">Follow Us</h5>
-                    <a href="#" target="_blank"><i class="fa fa-github p-2 text-white" aria-hidden="true"
-                            style="background:#3caea3; border-radius: 50%;"></i>
+                    <a href="#" target="_blank"><i id="follow"class="fa fa-github p-2 " aria-hidden="true"
```

Comment:
According to HTML standards the id should be unique in the page change that to a class and make the name more meaningful like `social-icon`.

Reply:
Add the github repo link as it is already present for Algo Phantoms.